### PR TITLE
Update canarieAPI doc links

### PR DESCRIPTION
**Pull request by huard**
- Updated components' version number. 
- Replaced links to githubio docs to readthedocs.
- renderer is provided by THREDDS-WMS. 
- slicer is provided by finch.

### CODE REVIEW

**Review by tlvu — CHANGES_REQUESTED** [review]
Use templating variable, else all good.

File: birdhouse/config/canarie-api/docker_configuration.py.template [hunk 1]
```diff
@@ -173,64 +173,64 @@ SERVICES = {
...
-            'licence': 'https://ouranosinc.github.io/pavics-sdi/license.html',
+            'licence': 'https://github.com/Unidata/tds/blob/master/LICENSE',
             'provenance': 'https://ouranosinc.github.io/pavics-sdi/provenance/index.html'
         },
         'monitoring': {
-            'ncWMS': {
+            'TDS-WMS': {
                 'request': {
-                    'url': 'https://${PAVICS_FQDN_PUBLIC}/ncWMS2/wms?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.3.0'
+                    'url': 'https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/wms/birdhouse/testdata/ta_Amon_MRI-CGCM3_decadal1980_r1i1p1_199101-200012.nc?service=WMS&version=1.3.0&request=GetCapabilities'
```
[the remark this answers is not in the document]
Comment (huard) [reply birdhouse/config/canarie-api/docker_configuration.py.template:200]: TWITCHER_PROTECTED_PATH is not used everywhere at the moment.

[the remark this answers is not in the document]
Comment (huard) [reply birdhouse/config/canarie-api/docker_configuration.py.template:200]: Shoud I replace `twitcher/ows/proxy` by `${TWITCHER_PROTECTED_PATH}` everywhere ?

[the remark this answers is not in the document]
Comment (tlvu) [reply birdhouse/config/canarie-api/docker_configuration.py.template:200]: Was introduced by Francis here https://github.com/bird-house/birdhouse-deploy/blob/fd8d811748577a52a92182682a6a30f79a35fc0b/birdhouse/env.local.example#L15 without migrating all references.

Anyways, it's still a good idea to centralize to that var and we started to use it in many places, when we touch related code, see https://github.com/bird-house/birdhouse-deploy/search?q=TWITCHER_PROTECTED_PATH

[the remark this answers is not in the document]
Comment (tlvu) [reply birdhouse/config/canarie-api/docker_configuration.py.template:200]: > Shoud I replace twitcher/ows/proxy by ${TWITCHER_PROTECTED_PATH} everywhere ?

Yes, careful to not introduce any typos :D

**Comment by tlvu** [comment]
run tests

**Comment by huard** [comment]
run tests

**Comment by huard** [comment]
run tests

**Comment by huard** [comment]
run tests

**Comment by tlvu** [comment]
run tests

**Review by tlvu — APPROVED** [review]

**Comment by tlvu** [comment]
@huard Jenkins passed https://daccs-jenkins.crim.ca/job/DACCS-iac/268/

**Pull request merged**